.github/workflows/rust.yml

```
@@ -8,94 +8,120 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  RUSTFLAGS: '-D warnings'
+  RUSTDOCFLAGS: '-D warnings'
 
 jobs:
-  test-u32:
-    name: Test u32 backend
+  test:
+    name: Test all features
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          # 32-bit target
+          - target: i686-unknown-linux-gnu
+            deps: sudo apt update && sudo apt install gcc-multilib
+          # 64-bit target
+          - target: x86_64-unknown-linux-gnu
     steps:
-    - uses: actions/checkout@v2
-    - uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        toolchain: stable
-        override: true
-    - uses: actions-rs/cargo@v1
-      with:
-        command: test
-        args: --no-default-features --features "std u32_backend"
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@stable
+      - run: rustup target add ${{ matrix.target }}
```

```suggestion
      - uses: dtolnay/rust-toolchain@stable
      - uses: dtolnay/rust-toolchain@stable
         with:
           target: ${{ matrix.target }}
```

---

`- uses: dtolnay/rust-toolchain@stable is doubled` ^^ ?

```suggestion
      - uses: dtolnay/rust-toolchain@stable
        with:
          target: ${{ matrix.target }}
```

right ?

---

Thanks fixed :)